src/pages/guidelines/color/overview.mdx

```diff
@@ -242,11 +267,14 @@ light themes.
 
 <br />
 
-#### High contrast
+### High contrast
 
 In some cases, it is helpful to apply light components to dark backgrounds or
 dark components to light backgrounds. This technique is useful to focus
-attention or create visual tension.
+attention or create visual tension. Some high contrast moments are backed into
+the themes used `inverse` tokens, like with the tooltip component. Other times
```

Comment:
Used seemed out of place in this sentence, but I could be reading it wrong. Maybe through inverse tokens, with inverse tokens, etc.

```suggestion
the themes used `inverse` tokens, like with the tooltip component. Other times
```